<launch>

<!--  <node pkg="topic_tools" type="throttle" name="throttle_joint_states" args="messages joint_states 0.5" /> -->

  <node pkg="sbpl_two_arm_planner_node" type="requestPlan" name="request_plan" output="screen" respawn="false" >

    <param name="start_x" value="0.7" />
    <param name="start_y" value="0.04" />
    <param name="start_z" value="0.94" />
    <param name="start_r" value="0.0" />
    <param name="start_p" value="0.0" />
    <param name="start_y" value="0.0" />

    <param name="goal_x" value="0.7" /> <!-- Plans to 0.64 in 4.2 seconds -->
    <param name="goal_y" value="-0.1" />
    <param name="goal_z" value="0.48" />
    <param name="goal_roll" value="0.0" />
    <param name="goal_pitch" value="0.0" />
    <param name="goal_yaw" value="0.0" />

    <param name="raise_torso" value="false" />

    <param name="right_arm_pose_on_object_x" value="0.00" />
    <param name="right_arm_pose_on_object_y" value="-0.15" />
    <param name="right_arm_pose_on_object_z" value="0.0" />

    <param name="left_arm_pose_on_object_x" value="0.0" />
    <param name="left_arm_pose_on_object_y" value="0.1" />
    <param name="left_arm_pose_on_object_z" value="0.0" />

  </node>

</launch>
